UiKitDZ/ReaderViewController.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  ViewController.swift
+//  UiKitDZ
+//
+//  Created by Александр Андреевич Щепелин on 21.09.2022.
+//
+
+import UIKit
+
+/// Экран с текствью фичами по шрифту и т.д
+final class ReaderViewController: UIViewController {
+    // MARK: - Enum
+    private enum Constants {
+        static let textForReader = """
+Земную жизнь пройдя до половины,
+Я очутился в сумрачном лесу,
+Утратив правый путь во тьме долины
+Каков он был, о, как произнесу,
+Тот дикий лес, дремучий и грозящий,
+Чей давний ужас в памяти несу!
+
+Так горек он, что смерть едва ль не слаще.
+Но, благо в нем обретши навсегда,
+Скажу про все, что видел в этой чаще.
+
+Не помню сам, как я вошел туда,
+Настолько сон меня опутал ложью,
+Когда я сбился с верного следа.
+"""
+        static let themeLabelText = "Light/Dark theme"
+    }
+    
+    // MARK: - Visual Components
+    
+    private lazy var sliderForFont: UISlider = {
```

Comment:
```suggestion
    private lazy var sliderForFont: UISlider = {
```
По названию это шрифт а по типу слайдер что то тут не так